lib/jsdom/browser/Window.js

```diff
@@ -193,6 +202,10 @@ function Window(options) {
           return this.responseText;
         } else if (this.responseType === "json") {
           return JSON.parse(this.responseText);
+        } else if (window.document.location.protocol === "file:") {
+          // node-xhr doesn't properly set the responseType for file requests.
+          // return the text anyway.
```

Comment:
This doesn't seem like correct fallback behavior. responseType is set by authors. You don't get to decide that it always be text for files.